bin/build-mb/src/build/licenses.clj

```diff
@@ -236,3 +226,21 @@
         ;; best defaults. Want to make sure we never kill our build script
         #_(System/exit (if (seq without-license) 1 0))))
     license-info))
+
+(comment
```

Comment:
are we doing the hickey-style comment dealios as A Thing now? it seems useful enough

Reply:
I left this one here so they can see how to interact with this if needed. If you aren't familiar with the basis this would let you quickly see what it is and shows how you can get a small subset.